src/pages/publication/PublicationPage.tsx

```diff
@@ -3,8 +3,44 @@ import { getPublication } from '../../api/publicationApi';
 import { addNotification } from '../../redux/actions/notificationActions';
 import i18n from '../../translations/i18n';
 import { useDispatch } from 'react-redux';
-import { CircularProgress } from '@material-ui/core';
+import { CircularProgress, Link } from '@material-ui/core';
 import { emptyPublication, Publication } from '../../types/publication.types';
+import styled from 'styled-components';
+import ContentPage from '../../components/ContentPage';
+import FormCardHeading from '../../components/FormCard/FormCardHeading';
+import FormCard from '../../components/FormCard/FormCard';
+import { useTranslation } from 'react-i18next';
+import LabelContentRowForPublicationPage from './publication_page/LabelContentRowForPublicationPage';
+import PublicationPageAuthors from './publication_page/PublicationPageAuthors';
+import PublicationPageFiles from './publication_page/PublicationPageFiles';
+import PublicationPageJournal from './publication_page/PublicationPageJournal';
+import PublicationPageIdentifiers from './publication_page/PublicationPageIdentifiers';
+import NormalText from '../../components/NormalText';
+import PublicationPageSeries from './publication_page/PublicationPageSeries';
+import NotFound from '../errorpages/NotFound';
+
+const ContentWrapper = styled.div`
+  display: flex;
+  padding-top: 1rem;
+
+  @media only screen and (max-width: ${({ theme }) => theme.breakpoints.values.sm + 'px'}) {
```

Comment:
vil vi ikke at dette også skal gjelde for ikke-screen, som ved print feks?

Reply:
good point

Reply:
(copy paste)

Reply:
fjerner i hele applikasjonen med det samme